intro-to-git/activity-git-history.md

```diff
@@ -0,0 +1,93 @@
+# Activity: git History
+
+<!-- Can be async -->
+<!-- Written to be synchronous -->
+<!-- Ideal Format: Small groups, large groups, roundtables, etc. -->
+
+## Goal
+
+Our goal is to practice the process of making commits as a group. We will also practice collaborating in Git as a group.
+
+Our goal is to practice this without actually typing and running these commands in the terminal; however, we should practice recognizing what commands we _would_ type.
+
+## Activity Instructions
+
+1. Assemble into small groups
+1. Read through the instructions of Part 1
+1. Do Part 1
+1. Read through the instructions of Part 2
+1. Do Part 2
+
+### Preparation
+
+Part 1 and Part 2 will be a combination of discussion and individual writing:
+- Each individual should have note-taking materials
+- Bring up any Git references, resources, lessons, and notes
+
+## Part 1
+
+Each group member will describe the steps to create a specific commit, and diagram what Git "looks like" at each step.
+
+Each member will pick one of the commits in the Setup section. These specific commits already have these details:
+- The number and names of files that contain changes
+- The commit message
+- The commit hash 
+
+The group members will use these details to name the specific commands needed.
+
+### Setup
+
+1. Begin a diagram of Git's areas. Each individual should draw a...
+        - box labeled "Untracked Changes Area"
+        - box labeled "Local Changes Area"
+        - box labeled "Staging Area"
+        - box labeled "Git Log"
+
+1. Each individual should pick one of these commits:
```

Comment:
Should there be more after this bullet? If not, let's remove the `:` at the end of the line.

Reply:
WOOPS LOL yes, totally correct y'all!!!! Thanks for catching on.

Gonna have a table of some mock commit hashes and their commit messages.